part.go

```diff
@@ -160,11 +160,23 @@ func (p *Part) setupContentHeaders(mediaParams map[string]string) {
 	}
 }
 
+func (p *Part) readPartContent(r io.Reader, readPartErrorPolicy ReadPartErrorPolicy) ([]byte, error) {
+	buf, err := ioutil.ReadAll(r)
+	if err != nil {
+		if readPartErrorPolicy != nil && readPartErrorPolicy(p, err) {
+			p.addWarning(ErrorMalformedChildPart, "keeping the buffer read on error: %s", err.Error())
```

Comment:
Text of this error doesn't make a lot of sense, could we perhaps call it a partial buffer/content/result/etc?

Reply:
Do you mean to create a new error const? Something like this:
```go
ErrorPartialContent := "Partial content" // this goes in error.go
p.addWarning("Partial content", "keeping the buffer read on error: %s", err.Error())
```

Or to just change the detail parameter?
```go
p.addWarning(ErrorMalformedChildPart, "partial content returned: %s", err.Error())
```

Reply:
Just the detail parameter text.  We do have an ErrorMalformedBase64 const already, but I'm assuming this error policy could apply to more scenarios than just base64?

Reply:
Exactly. I'll change then the detail parameter name. Thanks!